content/doc/book/pipeline/index.adoc

```diff
@@ -129,6 +129,9 @@ utilizing Freestyle Jobs such as the Copy Artifact, Parameterized Trigger, and
 Promoted Builds plugins] Pipeline makes this concept a first-class citizen in
 Jenkins.
 
+.What is the difference between Freestyle and Pipeline in Jenkins
+video::IOUm1lw7F58[youtube,width=800,height=420]
```

Comment:
Can we update the cc with punctuation and correct spelling errors?

Reply:
Unfortunately that wouldn't be possible at the moment, as the transcriptions require more resources and time than available. This may be something we can revisit in the future though, since having everything be uniform would be ideal.